projects/angular/src/utils/list-key-navigation/providers/focusable-items.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { InjectionToken, ValueProvider } from '@angular/core';
+
+export const FocusableElementsToken = new InjectionToken('FocusableElements');
+
+export const FocusableElements = {
```

Comment:
1. These constants should be renamed to snake case for consistency with other tokens/providers.
2. The provider constant should be named as such for consistency with other providers (end with "provider").
2. I think "elements" should be "element selectors" for code readability. (I was confused by `this.focusableElements` in the directive code b/c I thought it was an array of element rather than an array of selectors.)

So `FOCUSABLE_ELEMENT_SELECTORS_TOKEN` and `FOCUSABLE_ELEMENT_SELECTORS_PROVIDER`.